src/features/PeoplePage/index.js

```diff
@@ -59,6 +63,7 @@ const PeoplePage = () => {
                   }
                   title={person.name}
                 />
+                </StyledLink>
```

Comment:
probably you have check your code because StyledLInk you added to moviesPage, not moviePage. StyledLink has to be in moviesPage and PeoplePaga like I did, also have this fragment code in PeoplePage. If StyledLink was in ProfilePage , It wouldn't work. This component had different layout and I didn't change it. Styles are from different file and common file.

Reply:
I correct forrmatting

Reply:
I know, but my task was add redux to component, so component was creating a few days ago by Krystian, so if we decide which option we choose I change this component :)

Reply:
Ok, give redux to ProfilePage to keep it consistent. I also think that when we finish this part, it would be worth converting BigTile into one component (because we now have two in Tile and ProfilePage), I even have an idea for it. However, in order not to confuse now, we can do it later in a separate branch.

Reply:
We have two BigTile components because they differ significantly in the content of AdditionalInfo. When I was creating ProfilePage, I had no better idea and I'm just looking for a solution to this problem, because I think one BigTile is a better idea. Maybe it is worth asking Krzysztof Dąbrowski about it, maybe he will suggest some good solutions.

Reply:
Yes, I know, I have idea to Connect Two bigTiles into one with additional props. For example named profile. And then inside do the logic whitin in to generate diffrent content of AdditionalInfo. I will prepare it and add PullRequest.

Reply:
Iza, you are amazing,
before I commented on PR you merged the code into develop.
Good job 👏👏👏

Reply:
Thank you 😉 I' ve merged the code so fast, beacuse Kinga needs it to do ProfilePage 😀